Meta: Configure prettier

MarkusAbtion opened the pull request:
Add the prettier linter for javascript to the project.

For now, we've decided to use the usual `webpacker` folder structure for the javascript files, but based on a [recent slack discussion](https://abtion.slack.com/archives/CPW7LUT8B/p1598862193003800), it seems that the structure might change in the future.

## Asana task:
[Add configuration file for prettier, and ensure that files are prettified in our CI](https://app.asana.com/0/1142794766483633/1157388878186048/f)

jeppester commented:
> Add the prettier linter for javascript to the project.
> 
> For now, we've decided to use the usual `webpacker` folder structure for the javascript files, but based on a [recent slack discussion](https://abtion.slack.com/archives/CPW7LUT8B/p1598862193003800), it seems that the structure might change in the future.
> 
> ## Asana task:
> [Add configuration file for prettier, and ensure that files are prettified in our CI](https://app.asana.com/0/1142794766483633/1157388878186048/f)

As a result of that discussion we moved the FE files into `/frontend`. Does that matter to this PR?

jeppester commented:
I can see that I overlooked it. My mistake

jeppester commented on bin/prettier:
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+npx prettier@2.0.5 app/javascript $@
```
```suggestion
npx prettier@2.0.5 frontend $@
```

MarkusAbtion commented:
> > For now, we've decided to use the usual `webpacker` folder structure for the javascript files, but based on a [recent slack discussion](https://abtion.slack.com/archives/CPW7LUT8B/p1598862193003800), it seems that the structure might change in the future.
> 
> As a result of that discussion we moved the FE files into `/frontend`. Does that matter to this PR?

That was our mistake. We tried to look in other projects to see how it'd been implemented earlier. But we didn't check Muffi itself.

Fixed :)

jeppester reviewed (approved):
:+1:
